src/Microsoft.Health.Dicom.Functions/host.json

```diff
@@ -13,5 +13,29 @@
       "batchSize": 1000,
       "parallelBatches": 10
     }
+  },
+  "BlobStore": {
+    "ConnectionString": null,
+    "RequestOptions": {
+      "ExponentialRetryBackoffDeltaInSeconds": 4,
+      "ExponentialRetryMaxAttempts": 6,
+      "ServerTimeoutInMinutes": 2,
+      "DownloadMaximumConcurrency": 5,
+      "UploadMaximumConcurrency": 5
+    }
+  },
+  "SqlServer": {
+    "Initialize": "false",
+    "AllowDatabaseCreation": "false",
+    "ConnectionString": "server=(local);Initial Catalog=Dicom;Integrated Security=true",
+    "TransientFaultRetryPolicy": {
```

Comment:
my best guess is we only need `ConnectionString` and `TransientFaultRetryPolicy` (could be useful later) here, since we don't initialize database.

Reply:
True!